Features/Lookuptable/testPages/data/lookup_table_page.py

```diff
@@ -0,0 +1,771 @@
+        time.sleep(3)
+        self.wait_to_click(self.closedownloadpopup)
+
+    def latest_download_file(self):
+        os.chdir(PathSettings.DOWNLOAD_PATH)
+        files = sorted(os.listdir(os.getcwd()), key=os.path.getmtime)
+        max_file = max(files, key=os.path.getctime)
+        path = os.path.join(PathSettings.DOWNLOAD_PATH, max_file)
+        return path
+
+    def create_download_lookuptable(self):
+        self.create_lookup_table()
+        self.download1()
+        return self.table_id_name
+
+    def write_data_excel(self,table_id, path,):
+        excel = ExcelManager(path)
+        for x in UserData.col_headers:
+            for i in range(1, 2):
+                col = excel.col_size(table_id)
+                excel.write_excel_data(table_id, 1, col + i, x)
+            excel.write_data(table_id, UserData.data_list1)
+            excel.write_data(table_id, UserData.data_list2)
+
+    def update_excel_user_value(self,table_id,path):
+        excel = ExcelManager(path)
+        col = excel.col_size(table_id)
+        excel.write_excel_data(table_id, 1, col + 1, "user 1")
+        excel.upload_to_path(table_id, UserData.data_list)
+
+    def update_excel_group_value(self,table_id,path):
+        excel = ExcelManager(path)
+        col = excel.col_size(table_id)
+        excel.write_excel_data(table_id, 1, col + 1, "group 1")
+        excel.upload_to_path(table_id, UserData.data_list)
+
+    def compare_excel(self,df1,df2,compareflag):
+        if(compareflag ==0):
+            assert df1 == df2
+            print("Data is same")
+        else:
+            assert df1 != df2
+            print("Data is different in both excels")
+
+    def download_update_7(self, tableid, path):
+        excel = ExcelManager(path)
+        df1 = excel.read_excel(tableid)
+        self.err_upload(path)
+        df2 = excel.read_excel(tableid)
+        return df1, df2
+
+    def download_update_8(self,path,table_id):
+        excel = ExcelManager(path)
+        col = excel.col_size(table_id)
+        excel.write_excel_data(table_id, 1, col + 1, "user 1")
+        excel.write_excel_data(table_id, 1, col + 2, "group 1")
+        excel.write_data(table_id, UserData.data_list1)
+        self.err_upload(path)
+        d1 = excel.read_excel(table_id)
+        print("data1", d1)
+        excel.write_excel_data(table_id, 2, 4, 'kiran')
+        excel.write_excel_data(table_id, 2, 5, '123')
+        self.err_upload(path)
+        d2 = excel.read_excel(table_id)
+        print("data2", d2)
+        return d1, d2
+
+    def replace_existing_table(self, filepath):
+        self.wait_to_click(self.manage_tables_link)
+        self.scroll_to_bottom()
+        self.send_keys(self.upload_table, filepath)
+        self.wait_to_click(self.replace_table)
+        self.wait_to_click(self.upload)
+        time.sleep(20)
+        self.wait_to_click(self.manage_tables_link)
+
+
+    def delete_row_from_table(self, downloadpath, tablename):
```

Comment:
why are we passing 'downloadpath' as a parameter? This parameter is not used anywhere inside the function.